README.md

```diff
@@ -482,7 +513,7 @@ call.enqueue(new GraphCall.Callback<Storefront.QueryRoot>() {
 
 ## Card Vaulting [⤴](#table-of-contents)
 
-The Buy SDK offers support for native checkout via GraphQL, which lets you complete a checkout with a credit card. However, it doesn't accept credit card numbers directly. Instead, you need to vault the credit cards via the standalone, PCI-compliant web service. The Buy SDK makes it easy with `CardClient`.
+The Buy SDK offers support for native checkout via GraphQL, which lets you complete a checkout with a credit card. However, it doesn't accept credit card numbers directly. Instead, you need to vault the credit cards via the standalone, PCI-compliant web service. The Buy SDK makes it easy to do this using `CardClient`.
```

Comment:
the iOS docs have this as `Card.Client` instead of `CardClient` -- should they be consistent, or are they different?